HLTrigger/Configuration/python/customizeHLTTrackingForPhaseI2017.py

```diff
@@ -27,6 +27,9 @@ def modifyHLTPhaseIPixelGeom(process):
 
 # modify the HLT configuration to run the Phase I tracking in the particle flow sequence
 def customizeHLTForPFTrackingPhaseI2017(process):
+    if not hasattr(process, 'hltPixelLayerTriplets'):
+        #there could also be a message here that the call is done for non-HLT stuff
+        return process;
```

Comment:
yes, but it can't customise something that doesn't already exist in the HLT part. That's why this "if hasattr" is here